ChocolateChipCookieRecipe.md

```diff
@@ -15,4 +15,4 @@
 2. Cream together the butter and sugar together until light and fluffy (about 3 minutes).
 3. Whisk together the cornstarch, baking powder, and water until smooth. Mix into the creamed batter.
 4. Stir in the flour then the chocolate chips until combined.
-5. Bake until just lightly browned on the edges and middle still looks undercooked.
+5. Bake for 15-20 mins until just lightly browned on the edges and middle still looks undercooked.
```

Comment:
Requested change: Make 20-30 minute bake time. Our company ovens are kinda old and slow to heat.